.github/workflows/compliance.yml

```diff
@@ -94,16 +94,23 @@ jobs:
           exit 1;
         fi
 
+        warns=("ClangFormat")
         files=($(./scripts/ci/check_compliance.py -l))
+
         for file in "${files[@]}"; do
```

Comment:
@pdgendt could you look at making sure the clangformat check only runs on actual C and header files? see https://github.com/zephyrproject-rtos/zephyr/pull/77093/files#annotation_25086099623

Reply:
Created #77095